test/utils/image/manifest.go

```diff
@@ -30,6 +30,7 @@ import (
 
 // RegistryList holds public and private image registries
 type RegistryList struct {
+	AlpineRegistry          string `yaml:"alpineRegistry"`
```

Comment:
Where is this registry used?

Reply:
Ooops. This can be removed. It was used when doing the proxying via an alpine socat image, but then that got replaced by the proxy code in csi-test.